app/pages/cif/projects.tsx

```diff
@@ -52,7 +52,10 @@ export const ProjectsQuery = graphql`
         projectManagersByProjectId: {
           some: {
             cifUserByCifUserId: {
-              fullName: { includesInsensitive: $projectManagers }
+              fullName: { includesInsensitive: $primaryProjectManager }
+            }
+            projectManagerLabelByProjectManagerLabelId: {
+              label: { includesInsensitive: "primary" }
             }
           }
```

Comment:
I don't know what the default behaviour is for graphql, but it wouldn't hurt to specify the `and` logic between both conditions:
```graphql
some: {
   and: {
    cifUserByCifUserId:{...}
    projectManagerLabelBy...
  }
}
```